scripts/syschdemd.sh

```diff
@@ -111,7 +111,12 @@ main() {
 
   # If we're executed from inside the container, e.g. sudo
   if is_in_container; then
-    exec $command
+    # if the command already starts with an exec it must be wrapped in a shell
+    if [[ $command == "exec "* ]]; then
+      exec sh -c "$command"
```

Comment:
Does `eval` work here as well? That would get around the double exec

Reply:
That works, too